Public/Invoke-Locksmith.ps1

```diff
@@ -163,14 +163,9 @@ function Invoke-Locksmith {
     $SafeObjectTypes = '0e10c968-78fb-11d2-90d4-00c04f79dc55|a05b8cc2-17bc-4802-a710-e7c15ab866a2'
 
     <#
-        -512$ = Domain Admins group
         -519$ = Enterprise Admins group
-        -544$ = Administrators group
-        -18$  = SYSTEM
-        -517$ = Cert Publishers
-        -500$ = Built-in Administrator
     #>
-    $SafeOwners = '-512$|-519$|-544$|-18$|-517$|-500$'
+    $SafeOwners = '-519$'
```

Comment:
This seems significant. Can you elaborate?

Reply:
All items in the Public Key Services (PKS) should be owned by Enterprise Admins because the Configuration partition appears identically in all domains in a forest.

If this isn't applied it can be possible for domain to domain compromise or even domain to forest compromise. (I realize an attacker with control of any Tier 0 object can eventually compromise the forest regardless of this ownership, but let's put in speedbumps where we can!)

Reply:
Awesome insight. 💯